### PR TITLE
Style eligibility result pages

### PR DESCRIPTION
**Pull request by frederike-ramin**
# Short Description
- Style the eligibility result pages according to Figma.
- However, I made two changes that I also discussed with Caro: 1. the icons are 75 instead of 100 pixels wide, it just looked way better to us and 2. I did not extend the box over the base.html's right border of col-lg-9, because we never do this in any other step and changing the base html just for one step, that also works without changing the base just doesn't seem a good idea or worth the effort for me. You may disagree^^

# Changes
- Make icons smaller svgs
- Add padding below icons to have a better look on mobile
- Adapt cols and rows
- Reformat one file (sorry!)

# Feedback
- There was no mobile version provided and I'm not a 100% on how I handled that, so maybe you wanna have a look at that?

### CODE REVIEW

**Review by JulianRoesner — CHANGES_REQUESTED** [review]
> Reformat one file (sorry!)
Sorry for what?

> There was no mobile version provided and I'm not a 100% on how I handled that, so maybe you wanna have a look at that?

Yes, I think we need to discuss this with Nadine because the old version of this page had specifics for the mobile versions. So far it looks good to me.

File: webapp/app/static/css/eligibility.css [hunk 1]
```diff
@@ -18,12 +18,16 @@
     margin-bottom: var(--spacing-07);
 }
 
+.failure-success-icon {
+    min-width: 100px;
```
[the remark this answers is not in the document]
Comment (frederike-ramin) [reply webapp/app/static/css/eligibility.css:22]: Good point, I forgot to change it, will add padding instead. :)

File: webapp/app/templates/eligibility/display_success.html [hunk 2]
```diff
@@ -8,14 +8,16 @@
 {% endblock %}
 
 {% block step_content %}
-    <div class="section-intro">
-        <div class="headline row p-0">
```
[the remark this answers is not in the document]
Comment (frederike-ramin) [reply webapp/app/templates/eligibility/display_success.html:12]: I thought I didn't need the class anymore. In fact, I do, so I re-added it, but with different spacings. :)

File: webapp/app/templates/eligibility/display_success.html [hunk 3]
```diff
@@ -14,8 +14,8 @@
                 <img src="{{ url_for('static', filename='icons/success_icon.svg') }}" alt="{{ _('success_icon') }}"/>
             </div>
             <div class="col-10">
-                <h1>{{ render_info.step_title }}</h1>
-                <p class="result-text">{{ error_text }} {{ render_info.step_intro }}</p>
+                <h1 class="headline">{{ render_info.step_title }}</h1>
+                <p class="result-text m-0">{{ error_text }} {{ render_info.step_intro }}</p>
```
Comment (JulianRoesner) [inline webapp/app/templates/eligibility/display_success.html:18]: I am wondering, why do we need the m-0 here and do not set the margin directly in the result-text?

**Review by JulianRoesner — APPROVED** [review]
LGTM

File: webapp/app/templates/eligibility/display_success.html [hunk 4]
```diff
@@ -8,14 +8,16 @@
...
+                <img src="{{ url_for('static', filename='icons/success_icon.svg') }}" alt="{{ _('success_icon') }}"/>
+            </div>
+            <div class="col-10">
+                <h1 class="headline">{{ render_info.step_title }}</h1>
+                <p class="result-text m-0">{{ error_text }} {{ render_info.step_intro }}</p>
+            </div>
         </div>
-        <span class="result-text">
-            <p>{{ render_info.step_intro }}</p>
-        </span>
```
Comment (JulianRoesner) [inline webapp/app/templates/eligibility/display_success.html:18]: I am not sure, if you've seen it but we should also put the additional information into their own section with their own headline ("Bitte beachten Sie").

**Review by JulianRoesner — CHANGES_REQUESTED** [review]

File: webapp/app/templates/eligibility/display_success.html (hunk 4 above)
Comment (frederike-ramin) [reply webapp/app/templates/eligibility/display_success.html:18]: Thanks for pointing that out! I completely overlooked that!

**Review by JulianRoesner — APPROVED** [review]
👍

**Pull request merged**